homework_quiz2/README.md

```diff
@@ -0,0 +1,28 @@
+# Quiz Game
+
+## In order to launch a quiz game. You need to take a few steps.
+## The first step is to execute these commands.
+
+    git clone git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+    git checkout krylova
+
+## The second step. You need to go to the "homework_quiz2" folder. 
+    cd homework_quiz2
+
+## Create a container using this command. Fill in the database
+
+    chmod +x init_db.sh
+    ./init_db.sh 
```

Comment:
init script doesn't work
`./init_db.sh 
"docker run" requires at least 1 argument.
See 'docker run --help'.

Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Create and run a new container from an image
./init_db.sh: line 5: --name: command not found
./init_db.sh: line 6: -e: command not found
./init_db.sh: line 7: -e: command not found
./init_db.sh: line 8: -e: command not found
./init_db.sh: line 9: -v: command not found
./init_db.sh: line 10: -p: command not found
./init_db.sh: line 11: postgres:15.1: command not found
psql: error: could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5444?
psql: error: could not connect to server: Connection refused
	Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 38748?
`